Parsers/DuoSecurity/DuoSecurityAuthentication.txt

```diff
@@ -0,0 +1,44 @@
+// Title:           Duo Security Authentication Parser
+// Author:          Microsoft
+// Version:         1.0
+// Last Updated:    02/11/2021
+// Comment:         Inital Release
+//  
+// DESCRIPTION:
+// This parser takes Duo Security Authentication records ingested from the Function App Data Connector and parses the data into a normalized schema
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    It is recommended to name the Function Alias, as DuoSecurity<Log>
```

Comment:
Give proper function alias to save parser.[All parser]

Reply:
@v-maudan  do you have an example?

Reply:
@v-jayakal the parsers are not currently used in the workbooks or analytics.  They are available if the customer wants to use them